benchmarks/results/parquet.md

```diff
@@ -0,0 +1,25 @@
+
+
+## Parquet Benchmark
+
+1Fork 1Thread
+
+[info] Benchmark                           Mode  Cnt     Score     Error  Units
+[info] ParquetReaderBenchmark.fromTask    thrpt    5  2191.529 ± 493.092  ops/s
+[info] ParquetReaderBenchmark.unsafe      thrpt    5  2708.631 ± 508.417  ops/s
+[info] ParquetWriterBenchmark.fromCoeval  thrpt    5    98.009 ±  17.250  ops/s
+[info] ParquetWriterBenchmark.fromTask    thrpt    5    97.417 ±  15.160  ops/s
+[info] ParquetWriterBenchmark.unsafe      thrpt    5   100.652 ±   8.674  ops/s
```

Comment:
Looking at the benchmark numbers, both solutions (sefe and unsafe) seems to be approximated. They have run writing / reading files of 10 records.

Reply:
@Avasil finally removed `ParquetSink.fromWriter(parquetWriter: Task[ParquetWriter])` since providing `Coeval` should be enough, any additional comments/concerns?

Reply:
Great work 👍

Reply:
Great, thanks for helping! Will do a last review tonight before merge :)